docs/DeveloperGuide.md

```diff
@@ -453,23 +460,125 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Adding a friend
+1. Adding a friend to gitGud
 
-1. Deleting a person while all persons are being shown
+    1. Prerequisites: List all friends using the `friend --list` command. There should not be a friend with 
+       `FRIEND_ID` Draco already stored in gitGud.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
+    2. Test case: `friend --add Draco --name Marcus`<br>
+       Expected: Friend with `FRIEND_ID` Draco is added. gitGud states that `FRIEND_ID` is added.
+       
+    3. Test case: `friend --add MrFeely`<br>
```

Comment:
Maybe there be a prerequisite for MrFeely to not initially exist in the friends list as well?